static/game/deathscreen.js

```diff
@@ -61,6 +75,18 @@ export class DeathScreen extends Menu {
    * @override
    */
   draw() {
+    const cursor =
+      this.enteringUsername && this.cursorBlinkCounter % 100 < 50 ? "_" : " ";
```

Comment:
suppress default of the key events when entering the username. pressing space takes you to the bottom of the page (even if you don't allow space, it shouldn't do that)

Reply:
Fixed in 706c426